llvm_passes/ReplaceConstant.h

```diff
@@ -0,0 +1,59 @@
+//===- ReplaceConstant.h - Replacing LLVM constant expressions --*- C++ -*-===//
```

Comment:
IIUC, the sole user of ReplaceConstant is HipDynMem.cpp here:
```c++
  static void breakConstantExprs(const GVarVec &GVars) {
    for (GlobalVariable *GV : GVars) {
      for (Value *U : GV->users()) {
        ConstantExpr *CE = dyn_cast<ConstantExpr>(U);
        if (!CE) continue;
        SmallVector<Instruction*, 4> IUsers;
        getInstUsers(CE, IUsers);
        for (Instruction *I : IUsers) {
          convertConstantExprsToInstructions(I, CE);
        }
      }
    }
  }
```
Which seems to to the same thing as [convertUsersOfConstantsToInstructions()](https://github.com/llvm/llvm-project/blob/051aa171d2177776964cc309d14d8333af66135f/llvm/include/llvm/IR/ReplaceConstant.h#L25-L29). Can we use convertUsersOfConstantsToInstructions() instead of copying the ReplaceConstant sources?

Reply:
fixed